### DISCUSSION TITLE
mocap high level command for swarm

### DISCUSSION THREAD

**Discussion by mrfikri-ai** [bitcraze/discussions#910] [Q&A]
Hi,

I am just new to learning and using the Crazyflie. 
I tested the mocap hl-commander with Vicon, and it works well.
However, when I tried to extend the number of Crazyflies to more than 1.
Changing the URI like in hl-commander-swarm directly to mocap hl-commander directly was not good.
Do you know what should I do to be able to run multiple Crazyflies in hl-commander?

Thanks

**Comment by krichardsson** [comment]
I don't think we have an example for this, but in general terms you need to extend the hl-commander-swarm script to forward the positions that are measured by the mocap system to all the Crazyflies in the swarm (as is done in the mocap-hl-commander example).

You might want to try [Crazyswarm2](https://github.com/IMRCLab/crazyswarm2) as well, this is exactly what it is designed for :-)

**Reply by mrfikri-ai** [reply]
Hi

Thank you for the recommendation!
I will look at Crazyswarm2, while I try to deep more about the way to use mocap for Swarm.
